### ISSUE TITLE
Ability to see page(s) scanning schedule

### ISSUE THREAD

**Issue by yermulnik** [sneakypete81/updatescanner#184]
Would be handy to have an ability to see when it is scheduled to scan specific page(s) and ability to modify starting point of schedule.
I've got a lot of pages to be monitored which were migrated to FF57 all at once (considering as were added at the same time) and it sometimes looks for me like Update Scanner is scanning all of them simultaneously at some point in time (it literally influences FF responsiveness during this time), so I'd like to be able to change this behavior either by changing starting points of schedules or by adding e.g. some delay between each consecutive scans.

**Comment by einst3** [comment]
I think there should be a setting "Reset Clock" to reset all link clocks to restart update date from now.

**Comment by einst3** [comment]
I have always marked my links with [7-10im] so i know i check it every 7 days (1 week), 10 changes, ignore numbers (i), and manually (m). this way i know the schedule i applied.

**Comment by einst3** [comment]
it would be nice to see the next update date (no time) after/before the link.

**Comment by yermulnik** [comment]
Crap! It consumes all the memory and freezes my laptop when it runs update check =( Someone definitely has to re-factor this part of this great extension! pls!

**Comment by btanner2** [comment]
Would like to see this too

**Comment by sneakypete81** [comment]
A way of viewing when the next scan will occur would be useful, thanks for your input.

There are a few other feature requests above, could we split these into separate issues?

**Comment by yermulnik** [comment]
> There are a few other feature requests above, could we split these into separate issues?

Yup, of course, feel free to split those as much as it's handy for you to keep track of them.
